service/src/main/java/bio/terra/cbas/controllers/RunSetsApiController.java

```diff
@@ -177,16 +174,15 @@ public ResponseEntity<RunSetStateResponse> postRunSet(RunSetRequest request) {
         HttpStatus.OK);
   }
 
-  private static Optional<ResponseEntity<RunSetStateResponse>> checkInvalidRequest(
-      RunSetRequest request, int runSetsMaximumRecordIds) {
+  public static boolean requestIsValid(RunSetRequest request, CbasApiConfiguration config) {
     int recordIdsSize = request.getWdsRecords().getRecordIds().size();
-    if (recordIdsSize > runSetsMaximumRecordIds) {
+    int recordIdsMax = config.getRunSetsMaximumRecordIds();
+    if (recordIdsSize > recordIdsMax) {
       log.warn(
           "Bad user request: %s record IDs submitted exceeds the maximum value of %s"
```

Comment:
**Not for this PR**, just something to think about for the future... once we are able to return error messages back to the user in "failure" responses, how might we bring this message up so we can include it in the response we send back?